app/views/admin/registration_periods/show.html.haml

```diff
@@ -26,4 +26,11 @@
         method: :delete, data: { confirm: 'Are you sure?' }, class: 'btn btn-danger'
     - else
       - if can? :create, @conference.build_registration_period
-        = link_to 'New Registration Period', new_admin_conference_registration_period_path, class: 'btn btn-primary'
+        - unless @conference.registration_tickets.empty?
+          = link_to 'New Registration Period', new_admin_conference_registration_period_path, class: 'btn btn-primary'
```

Comment:
Although you don't show the link, the path would be still accessible and the admin could create a registration period without registration tickets.

And another concern, what is the admin do not want to make the registration compulsory?